Draft-Accepted/RFC0066-PowerShell-User-Content-Location.md

```diff
@@ -0,0 +1,133 @@
+---
+RFC: RFC0066
+Author: Justin Chung
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Core
+Comments Due: 07/31/2025
+Plan to implement: Yes
+---
+
+# PowerShell User Content Location
+
+This RFC proposes moving the current PowerShell user content location out of OneDrive &
+default to the `AppData` directory on Windows machines.
```

Comment:
If this is AppData then this would be `$env:AppData` which is the roaming profile location not the local profile one. Please correct  to `LocalAppData`